string target_name
---
bool existence
